ui.content/src/main/content/jcr_root/conf/asset-share-commons/settings/wcm/templates/search-template-dark/initial/.content.xml

```diff
@@ -34,7 +34,7 @@
                     eventId="cart-add"
                     style="success"
                     text="Asset added to cart"/>
-                <cart-exists
+                <cart-remove
```

Comment:
cart-exists is not part of the page properties dialog, and the text on this node corresponds with cart-remove